client.py

```diff
@@ -141,13 +149,13 @@ async def run_client():
                 try:
                     data = await get_latest_message(websocket)
                     if data == "1" and current_state == False:
-                        set_capslock_state(True)
-                        current_state = True
-                        last_state = True
+                        set_capslock_state(True != dissident)
```

Comment:
```suggestion
                        set_capslock_state(not dissident)
```

& etc for all of the below lines may be much more readable